x-pack/plugins/cases/public/components/case_view/components/case_view_activity.tsx

```diff
@@ -179,15 +197,42 @@ export const CaseViewActivity = ({
       ? [caseUsers.reporter]
       : [convertToCaseUserWithProfileInfo(caseData.createdBy)];
 
+  const handleUserActionsActivityChanged = useCallback(
+    (params: UserActivityParams) => {
+      setUserActivityQueryParams({ type: params.type, sortOrder: params.sortOrder });
```

Comment:
nit: To be future-proof we can do `setUserActivityQueryParams((oldParams) => ({...oldParams, ... params})`